pykern/pkunit.py

```diff
@@ -551,7 +551,7 @@ def _actual_xlsx_to_csv(self, actual_xlsx, sheet):
             str(self._actual_path),
             encoding="utf-8",
             index=False,
-            line_terminator="\r\n",
+            lineterminator="\r\n",
```

Comment:
this needs to be conditional:
`TypeError: NDFrame.to_csv() got an unexpected keyword argument 'lineterminator'`

Reply:
You mean we should support both versions of pandas?

Reply:
or force the version of pandas in pykern/setup.py and wherever else it is